client/task-list/tasks/PaymentGatewaySuggestions/components/WCPay/utils.js

```diff
@@ -50,22 +50,19 @@ export function installActivateAndConnectWcpay(
 }
 
 export function isWCPaySupported( countryCode ) {
-	const supportedCountries = [ 'US', 'PR' ];
-	if (
-		window.wcAdminFeatures &&
-		window.wcAdminFeatures[ 'wcpay/support-international-countries' ]
-	) {
-		supportedCountries.push(
-			'AU',
-			'CA',
-			'DE',
-			'ES',
-			'FR',
-			'GB',
-			'IE',
-			'IT',
-			'NZ'
-		);
-	}
+	const supportedCountries = [
```

Comment:
Just making a note here for the future that it would be great to share the same array between server and client to prevent these from getting out of sync.

But unrelated to this PR and I'd prefer this to be an endpoint as opposed to preloaded data.